cypress-custom/integration/fee.test.ts

```diff
@@ -1,62 +1,71 @@
 import { ChainId, WETH } from '@uniswap/sdk'
 
+const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
+const RINKEBY = ChainId.RINKEBY.toString()
 const FEE_QUERY = `https://protocol-rinkeby.dev.gnosisdev.com/api/v1/tokens/${WETH[4].address}/fee`
 const FEE_QUOTES_LOCAL_STORAGE_KEY = 'redux_localstorage_simple_fee'
 
+function _assertFeeData(fee: any): void {
```

Comment:
```suggestion
function _assertFeeData(fee: FeeInformation): void {
```

Reply:
well, I'll do the change, but I don't necessarily agree this is necessarily better, I mean the function responsibility is to assert something has a special shape, but it can accept anything (and then fail of course)

Anyways, it also works for me. Not strong oppinion.